domain/src/main/kotlin/no/nav/su/se/bakover/domain/grunnlag/SjekkOmGrunnlagErKonsistent.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.su.se.bakover.domain.grunnlag
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import arrow.core.separateEither
+
+data class SjekkOmGrunnlagErKonsistent(
+    private val uføregrunnlag: List<Grunnlag.Uføregrunnlag>,
```

Comment:
Bør vi heller ta inn Grunnlagsdata? Og på sikt også Vilkårsvurderinger

Reply:
Unngikk grunnlagsdata inntil videre da jeg oppfattet det som at du holdt på å fjerne bla. uføregrunnlag derfra. Valgte også å holde det på grunnlagsnivå for å slippe å deale med all kompleksiteten fra vilkårsvurderinger. Tenker vi bare endrer inputen etterhvert som vi finner ut hva som er hensiktsmessig?